xtable-core/src/main/java/org/apache/xtable/delta/DeltaConversionSource.java

```diff
@@ -99,11 +101,16 @@ public TableChange getTableChangeForCommit(Long versionNumber) {
     Snapshot snapshotAtVersion = deltaLog.getSnapshotAt(versionNumber, Option.empty());
     FileFormat fileFormat =
         actionsConverter.convertToFileFormat(snapshotAtVersion.metadata().format().provider());
-    Set<InternalDataFile> addedFiles = new HashSet<>();
-    Set<InternalDataFile> removedFiles = new HashSet<>();
+
+    // All 3 of the following maps use data file path as the key
```

Comment:
nitpick: there are two maps? maybe just saying `data structures` is better here. Can you add whether the paths are absolute or relative in this case as well?